low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -55,23 +55,19 @@ SSPL_LL_SETTING:
       - Smartctl
       - NodeHWactuator
       - RealStorActuator
-   degraded_state_modules:
-      - DiskMonitor
-      - ServiceMonitor
-      - RAIDsensor
-      - RAIDIntegritySensor
+   ALWAYS_ENABLE_SENSORS:
       - NodeData
       - IEMSensor
-      - NodeHWsensor
-      - DiskMsgHandler
-      - LoggingMsgHandler
-      - ServiceMsgHandler
-      - NodeDataMsgHandler
-      - NodeControllerMsgHandler
-      - RealStorActuatorMsgHandler
-      - SASPortSensor
-      - MemFaultSensor
-      - CPUFaultSensor
+      - ServiceMonitor
+   ENV_BASED_SENSORS:
+      - REALSTORSENSORS
+      - NODEHWSENSOR
```

Comment:
Also in general, dont repeat sensor names in different sections, unless there is unavoidable case, need justification.

Reply:
Not needed this list in confi, Removed.